codegen/package.go

```diff
@@ -69,6 +71,7 @@ func NewPackageHelper(
 	copyrightHeader string,
 	annotationPrefix string,
 	stagingReqHeader string,
+	traceKey string,
```

Comment:
No objection here, but feels like `PackageHelper` is out of control, we need to think about how to prevent it from growing endlessly.